src/app/ubs/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html

```diff
@@ -49,6 +49,7 @@ <h3 class="form__headline">{{ 'order-details.title' | translate }}</h3>
                 min="0"
                 max="999"
                 formControlName="actualQuantity{{ bag.id }}"
+                [attr.disabled]="isOrderCancelled && isOrderPaid ? true : null"
```

Comment:
You should use readonly instead attr.disabled

Reply:
Thanks, I have changed it to just `disabled`)